implementations/micrometer-registry-dynatrace/src/test/java/io/micrometer/dynatrace/v2/DynatraceExporterV2Test.java

```diff
@@ -224,33 +225,40 @@ void toFunctionCounterLine() {
 
         List<String> actual = exporter.toFunctionCounterLine(functionCounter).collect(Collectors.toList());
         assertThat(actual).hasSize(1);
-        assertThat(actual.get(0)).hasSize("my.function.counter,dt.metrics.source=micrometer count,delta=0 1617776498381".length());
+        assertThat(actual.get(0)).hasSize("my.function.counter,dt.metrics.source=micrometer count,delta=0.0 1617776498381".length());
         assertThat(actual.get(0)).startsWith("my.function.counter,dt.metrics.source=micrometer count,delta=");
     }
 
     @Test
     void toFunctionTimerLine() {
-        class TestClass {
-            long timeSomething() {
-                return 3L;
+        class FunctionTimerDummy implements FunctionTimer {
+            @Override
+            public double count() {
+                return 500.5;
             }
 
-            double doSomething() {
-                return 5.3;
+            @Override
+            public double totalTime(TimeUnit unit) {
+                return 5005;
             }
 
-            final TimeUnit unit = TimeUnit.MILLISECONDS;
-        }
-
-        TestClass tester = new TestClass();
-        FunctionTimer.builder("my.function.timer", tester, TestClass::timeSomething, TestClass::doSomething, tester.unit).register(meterRegistry);
-        FunctionTimer functionTimer = meterRegistry.find("my.function.timer").functionTimer();
-        assertNotNull(functionTimer);
+            @Override
+            public TimeUnit baseTimeUnit() {
+                return TimeUnit.MILLISECONDS;
+            }
 
-        List<String> actual = exporter.toFunctionTimerLine(functionTimer).collect(Collectors.toList());
+            @Override
+            public Id getId() {
+                return  new Id("my.function.timer", Tags.empty(), null, null, Type.TIMER);
+            }
+        }
+        
+        FunctionTimerDummy cut = new FunctionTimerDummy();
```

Comment:
Why `cut`?

Reply:
used to be `ClassUnderTest`, changed the class name but not the variable name